### PR TITLE
feat: fetch third party integrations from subgraph endpoint

### PR DESCRIPTION
**Pull request by guidota**
## Description

Adds `/third-party-integrations` endpoint.

It will query The Graph third party registry subgraph and return the list of integrations.

### CODE REVIEW

File: lambdas/src/Server.ts [hunk 1]
```diff
@@ -120,6 +121,8 @@ export class Server {
 
     // Functionality for Explore use case
     this.app.use('/explore', initializeExploreRoutes(express.Router(), env.getBean(Bean.DAO), contentClient))
+
+    this.app.use('/third-party-integrations', initializeThirdPartyIntegrationsRoutes(theGraphClient, express.Router()))
```
Comment (agusaldasoro) [inline lambdas/src/Server.ts:125]: Should we update the docs?

File: lambdas/src/apis/third-party/controllers/third-party.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { Response } from 'express'
+import { TheGraphClient } from '../../../utils/TheGraphClient'
+import { TimeRefreshedDataHolder } from '../../../utils/TimeRefreshedDataHolder'
+import { ThirdPartyIntegration } from '../../collections/types'
+
+let thirdPartyIntegrationsCache: TimeRefreshedDataHolder<ThirdPartyIntegration[]>
+
+export function initCache(theGraphClient: TheGraphClient): void {
+  thirdPartyIntegrationsCache = new TimeRefreshedDataHolder(() => fetchThirdPartyIntegrations(theGraphClient), '1m')
```
Comment (agusaldasoro) [inline lambdas/src/apis/third-party/controllers/third-party.ts:9]: Will we querying the graph every one minute? Is this really necessary? Also, can this be configurable?

Comment (guidota) [reply lambdas/src/apis/third-party/controllers/third-party.ts:9]: I think it's aligned with block times and graph indexing (more less). It is not actively updating this data every 1m, just holding it as caching strategy.

We can add timer configuration for sure.

Comment (agusaldasoro) [reply lambdas/src/apis/third-party/controllers/third-party.ts:9]: Okay, can we increase that default interval? Or do you think that it's a risk?

File: lambdas/src/Server.ts (hunk 1 above)
Comment (menduz) [reply lambdas/src/Server.ts:125]: Can we have a `@beta` channel in the docs? I'm afraid that people consider unstable APIs as part of the protocol and then those should be working forever after that

**Review by agusaldasoro — APPROVED** [review]

**Pull request merged**